.github/workflows/test-release-trigger.yml

```diff
@@ -12,12 +11,15 @@ jobs:
   trigger-circleci-release:
     runs-on: ubuntu-latest
     steps:
-
-      - name: Trigger Release job
+      - name: Release description
         if: github.event_name == 'workflow_dispatch'
         run: |
           echo "Triggering Test release using BRANCH and TEST_TAG ${{ github.event.inputs.branch }}"
+      - name: Trigger Release workflow in boundary-ui-releases
+        uses: benc-uk/workflow-dispatch@798e70c97009500150087d30d9f11c5444830385
```

Comment:
We might want to switch to using the built in `gh` CLI since this isn't in the TSCCR currently and it might not be approved. The change would look similar to what we do in the CRT [promote-production-helm](https://github.com/hashicorp/crt-workflows-common/blob/main/.github/workflows/promote-production-helm.yml#L84-L89. )

```
          export GITHUB_TOKEN="${ELEVATED_GITHUB_TOKEN}"
          gh workflow run release.yml \
            --repo hashicorp/boundary-ui-releases \
            --ref main \
            -f TAG="${{ github.event.inputs.branch }}"
```